### ISSUE TITLE
[Enhancement]: Add "Copy address" button in my account forms

### ISSUE THREAD

**Issue by senadir** [woocommerce/woocommerce#51647]
### Describe the solution you'd like

Right now, when you create an account via my account screens, the customer has no addresses. In the My Account -> Addresses screen, it would be nice if there's an option to copy an address from shipping to billing or vice versa.

![image](https://uploads.linear.app/f629fcde-6f9b-48e8-8af3-60eed8823bff/f22eec2e-e33c-412f-8033-6313055212a3/0d284ee3-33a9-47d5-8b6c-e9bf28e3cd23?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2Y2MjlmY2RlLTZmOWItNDhlOC04YWYzLTYwZWVkODgyM2JmZi9mMjJlZWMyZS1lMzNjLTQxMmYtODAzMy02MzEzMDU1MjEyYTMvMGQyODRlZTMtMzNhOS00N2Q1LThiNmMtZTliZjI4ZTNjZDIzIiwiaWF0IjoxNzgzNjAyNjcyLCJleHAiOjE4MTUxNzMyMzJ9.B_KIxC3GIwcewVm4TZ1hlTE2igQd2ZN6fUjaHFtUJqw)

### Describe alternatives you've considered

Write address again.<br>Hack a button to do so.

### Additional context

Forum ticket: [https://wordpress.org/support/topic/automatically-populate-shipping-address-in-customer-account/#post-18032702](<https://wordpress.org/support/topic/automatically-populate-shipping-address-in-customer-account/#post-18032702>)

**Comment by MKJJJ** [comment]
While this seems like a minor issue, it would be a significant improvement for many users. It wouldn’t hurt at all, since this dataset is filled in after the first order anyway. We added some text to make this clear for the customer. However, this needs a modification of the language file which is not a perfect solution.

**Comment by samrat-biswas-git** [comment]
Reviewed and confirmed this is still valid. Marking as ready for engineering.

Current evidence: My Account address editing still renders separate contact/address forms and there is no copy-from-billing or copy-from-shipping control on trunk. The issue already carries a design-feedback label, which matches the main blocker.

**Comment by JanaMW27** [comment]
How can someone create an account? I don't see an entry point to create an account. Am I missing something?

<img width="1491" height="967" alt="Image" src="https://github.com/user-attachments/assets/81c1f769-88d0-466f-ba04-62a64cb80d5a" />

**Comment by senadir** [comment]
@JanaMW27 you need to enable it from WooCommerce settings

<img width="932" height="236" alt="Image" src="https://github.com/user-attachments/assets/fa064a17-d028-41bb-9d5a-b7a727fec9a6" />

<img width="1101" height="739" alt="Image" src="https://github.com/user-attachments/assets/d8b54b23-f12c-436d-a5b0-071b2e2cd6da" />

**Comment by JanaMW27** [comment]
Thank you @senadir !

Here is my thought process on this one and where I landed.

**The problem I was solving**

A new account starts with no saved addresses, so in My Account > Addresses you fill in billing and shipping as two separate forms, even when they are the same address. First-time setup means typing the same thing twice.

**The options I weighed**

Two ways to build the copy:
- One click. You click “Copy Billing address,” it copies into shipping, saves, and reloads the page with the address in place.
- Copy into a prefilled form. Same click, but instead of saving it drops you on the edit form prefilled and unsaved, so you review and then hit Save.

**The email question that almost changed my mind**

I almost picked the prefilled form because of email. Billing has an email field and shipping does not, so copying shipping into billing looked like it would leave the email blank. It does not. When no billing email is saved, WooCommerce uses the account email, and the copy does the same. So the extra step was not needed.

**Where I landed**

One click. You click “Copy Billing address,” the address copies across, saves, and the page reloads with the shipping address in place and the normal “Edit Shipping address” link. If the two are not actually identical, Edit is right there to adjust. It works both directions.

In the empty column, “Copy Billing address” sits above the existing “Add Shipping address” link, and it would sit in the same position in the billing column with "Copy Shipping address" if the shipment address is the one that has been provided.

<img width="1558" height="1084" alt="Image" src="https://github.com/user-attachments/assets/5cbdf2b8-7e74-4dc8-b1e6-bb2773bcea69" />


Feedback welcome on the control label and on saving-on-click versus the prefilled-form alternative.

**Comment by MKJJJ** [comment]
Yes, please add this feature. Or even better: Add this feature AND a note stating that the shipping address does NOT need to be filled out if it matches the billing address. We did this using a simple function:

`add_action('woocommerce_my_account_after_my_address', 'custom_content_after_address_section');
function custom_content_after_address_section($address_type) {
    if ('shipping' === $address_type) {
            echo '<p class="shipping_address_hint">Sie müssen <strong>keine Lieferadresse</strong> angeben, wenn die <strong>Rechnungsadresse der Lieferanschrift</strong> entspricht. In diesem Fall wird die Lieferadresse automatisch nach der ersten Bestellung gespeichert.</p>';
    }
}`

The text translates to something like: You do not need to provide a <strong>shipping address</strong> if the <strong>billing address matches the shipping address</strong>. In this case, the shipping address will be saved automatically after your first order.
